assembly/config.json

```diff
@@ -3,6 +3,7 @@
         "docs-src/cloud",
         "docs-src/concepts",
         "docs-src/dev-guide",
+        "docs-src/dotnet",
```

Comment:
Question - is there any harm in referencing a dir that doesn't exist? Would it make more sense to wait until the PR that adds this dir to add this entry?